src/main/webapp/resources/js/pages/analysis/components/AnalysisTabularPreview.jsx

```diff
@@ -19,18 +20,25 @@ export function AnalysisTabularPreview({ output }) {
   const { firstLine, fileExt } = output;
   const fileExtCSV = fileExt === "csv";
 
-  const [fileRows, setFileRows] = useState([]);
-  const [fileCols, setFileCols] = useState([]);
+  const [fileRows, setFileRows] = React.useState([]);
+  const [fileCols, setFileCols] = React.useState([]);
   const MAX_TABLE_ROWS_PER_PAGE = 5;
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = React.useState(false);
+
+  const [total, setTotal] = React.useState(0);
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
+
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b